### DISCUSSION TITLE
Scans to collect before the first real-time neurofeedback session

### DISCUSSION THREAD

**Discussion by smweis** [OpenNFT/OpenNFT#155] [Q&A] [answered]
Hi all - we're looking forward to implementing OpenNFT and have had success so far running simulations on our system. The test at the scanner and acquiring 'real' data is coming up next. 

I see you have a T1 scan acquired and a motion_correction_template, which appears to be an EPI. There are also then ROIs provided in the subject's anatomical space. I'm wondering if you could provide some details about what these scans are, and what steps OpenNFT does exactly? 

I'm guessing the answers are the following: 

The motion_correction_template is an EPI scan that has already been registered to the T1 scan. The ROIs are provided in anatomical space (same as motion_correction_template now). When each scan comes in, the acquired EPI is registered (or transformed using an existing matrix?) to the motion_correction_template, then the ROIs are extracted? 

So, the day of neurofeedback, we should already have (or immediately acquire and process) 1 structural scan and 1 scout EPI scan, which immediately gets registered to structural space? 

Is all that correct? If not, what have I missed? If so, could some of this be added to the documentation?

**Comment by tiborauer** [comment, marked as answer]
Your approach is sensible if you are starting from anatomical ROIs. However, I suggest registering the T1 and the ROIs to the EPI, rather than the other way around. It save space (in memory) and avoid unnecessary upsampling.

If you use functional localiser, than the motion correction template is the mean EPI from the analysis of the functional localiser, and the ROIs comes from the analysis. Here, I also suggest registering T1 to EPI.
